index.js

```diff
@@ -14,6 +14,10 @@ const generatePyramid = (size, emoji) => {
     console.log("Pyramid can't be smaller than 2 emojis");
     return;
   }
+  if (!+size){
```

Comment:
🤔seems too ninja for me (and a lot of people i guess)

```javascript
if (isNaN(Number(size))) {
  // ...
}
```

Reply:
I knew you would say that when I wrote it 😅. I looks cool tho 👽